action.yml

```diff
@@ -1,6 +1,6 @@
-name: "Python setup uv"
```

Comment:
I "had" to name it this way because `setup uv` was already taken on the GitHub marketplace. If we don't want to publish there is should be fine

Reply:
Ahh interesting, ok. Let me think about this then.

Reply:
Is there any way to preview how this would render etc.?

Reply:
Other than looking at https://github.com/marketplace?type=actions I have no idea =/

Reply:
I realized that this action is no longer on the marketplace since the repo move.
But as an example you can have a look at how `setup-rye` renders: https://github.com/marketplace/actions/python-setup-rye

Reply:
I think this should work. Only thing missing is for the astral-sh org to accept some T&Cs so it can be published